File: SSNodeJsChat-Android/app/src/main/java/in/newdevpoint/ssnodejschat/utility/Watting.java
package in.newdevpoint.ssnodejschat.utility;

import android.content.Context;

import in.newdevpoint.ssnodejschat.R;


public class Watting extends android.app.ProgressDialog {
    public Watting(Context context) {
        super(context);
        setTitle(context.getString(R.string.loading));
        setCancelable(false);
    }
}

File: SSNodeJsChat-Android-Kotlin/app/src/main/java/in/newdevpoint/ssnodejschat/utility/Watting.kt
package `in`.newdevpoint.ssnodejschat.utility

import `in`.newdevpoint.ssnodejschat.R
import android.app.ProgressDialog
import android.content.Context

class Watting(context: Context) : ProgressDialog(context) {
    init {
        setTitle(context.getString(R.string.loading))
        setCancelable(false)
    }
}